charts/influxdb2/templates/ingress.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           service:
             name: {{ include "influxdb.fullname" . }}
             port:
-              number: 8086
+              name: {{ .Values.service.portName }}
 {{- else }}
           serviceName: {{ include "influxdb.fullname" . }}
           servicePort: 8086
```

Comment:
Could you please fix it here too. Thank you.

Reply:
I wasn't familiar with the v1beta1 spec, so I assumed it was correct there. I managed to track down the spec though and it also supports the service port name so I changed it there too.